web/src/utils/constants.ts

```diff
@@ -41,17 +41,17 @@ export enum TrackEvent {
 
 // Production/imports-exports mode
```

Comment:
Perhaps we can add a comment like this to explain a bit further how the colors were decided upon
```suggestion
// color of different production modes are based on various industry standards
```